.github/workflows/release.yml

```
@@ -14,8 +14,20 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
+      - name: Authenticate
+        id: app-token
+        uses: actions/create-github-app-token@v3
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v3". all required inputs are "app-id", "private-key"

(action)